plaid/plaid.go

```diff
@@ -1,373 +1,160 @@
-// Package plaid implements a Go client for the Plaid API (https://plaid.com/docs)
 package plaid
 
 import (
+	"bytes"
 	"encoding/json"
 	"errors"
+	"fmt"
 	"io"
 	"io/ioutil"
 	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/BurntSushi/toml"
 )
 
-// NewClient instantiates a Client associated with a client id, secret and environment.
-// See https://plaid.com/docs/api/#gaining-access.
-func NewClient(clientID, secret string, environment environmentURL) *Client {
-	return &Client{clientID, secret, environment, &http.Client{}}
+type PlaidEnvironment string
```

Comment:
Can we put this enum in a new file `environments.go`?

Reply:
fixed!